config/admins.txt

```diff
@@ -70,3 +70,4 @@ tribeofbeavers - Admin
 virtualjohn - Admin
 wjohnston - Admin
 wubli - Admin
+wilchen - Admin
```

Comment:
unless MSO has set up a separate server config this is the actual config that the server uses so this should probably go right out

Reply:
Dumb cat